frequently_asked_questions.md

```diff
@@ -0,0 +1,47 @@
+# FAQ
```

Comment:
[required] Let's link to this doc from other places, such as the main readme and troubleshooting docs.

Reply:
Sounds good to me